### ISSUE TITLE
refactor: remove id resolving

### ISSUE THREAD

**Issue by gamertike** [guildedts/guilded.ts#104]
### Package

guilded.ts

### Description

Remove useless id resolving. Id resolving is not needed in most manager methods as every structure instance already has methods that use its id for you. For example:

```js
message.edit(...);
// runs this
messages.edit(message.id, ...);
```

### Solution

Remove support for structure instances and only allow an ID as a parameter of manager methods.

**Comment by Pdzly** [comment]
Yeah the updated is better so that no duplicates / aliase are in the code
